packages/ui-causal-graph-editor/src/graph-viewer/causal-graph-editor.tsx

```diff
@@ -760,6 +791,9 @@ function CausalGraphEditor({ requireFocusToZoom = true, ...props }: CausalGraphE
                                 </>
                             }
                             topLeft={<RecalculateLayoutButton onResetLayout={resetLayout} />}
+                            loadingIndicator={
+                                isLayoutComputing && <LayoutSpinner text="Calculating layout..." size="24px" />
```

Comment:
Same reason here, the size I picked to align with the size with the top buttons so it doesn't look out of place

Reply:
Would it be better to have some height like the banner so the font-size changes but it stays aligned? Because the text in the banner will change size, or maybe that shouldn't change either?

Reply:
The font-size is 1.2rem atm, I'll probably just hard-code it for now to match the size.